src/neuroconv/datainterfaces/ophys/tdt_fp/tdtfiberphotometrydatainterface.py

```diff
@@ -273,6 +273,7 @@ def add_to_nwbfile(
         self,
         nwbfile: NWBFile,
         metadata: dict,
+        stub_test: bool = False,
```

Comment:
Note that this is a breaking change for anyone that was relying on the order of arguments in the conversion options.

Reply:
Good catch, another point in favor of kw-only args I would say.